src/pudl/metadata/fields.py

```diff
@@ -1257,31 +1561,77 @@
         "description": "Fuel content per unit of electricity generated. Calculated from FERC reported fuel consumption and net generation.",
         "unit": "MMBtu_MWh",
     },
-    "highest_distribution_voltage_kv": {"type": "number", "unit": "kV"},
-    "home_area_network": {"type": "integer"},
+    "highest_distribution_voltage_kv": {
+        "type": "number",
+        "description": "The highest voltage that's part of the distribution system.",
+        "unit": "kV",
+    },
+    "home_area_network": {
+        "type": "integer",
+        "description": (
+            "Number of AMI meters with home area network (HAN) gateway enabled."
+        ),
+    },
     "hrsg": {
         "type": "boolean",
         "description": "indicates if the boiler is a heat recovery steam generator (HRSG).",
     },
-    "inactive_accounts_included": {"type": "boolean"},
+    "inactive_accounts_included": {
+        "type": "boolean",
+        "description": (
+            "Whether the respondent includes inactive accounts in its definition of "
+            "customers used to determine SAIDI and SAIFI."
+        ),
+    },
     "income_type": {
         "type": "string",
         "description": "Type of income reported in income_statement_ferc1 table.",
     },
-    "income": {
-        "type": "number",
-        "description": "Utility income reported by income type.",
-        "unit": "USD",
-    },
     "increase_in_other_regulatory_liabilities": {
         "type": "number",
         "description": "The increase during the reporting period of other regulatory liabilities.",
         "unit": "USD",
     },
-    "incremental_energy_savings_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_life_cycle_energy_savings_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_life_cycle_peak_reduction_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_peak_reduction_mw": {"type": "number", "unit": "MW"},
+    "incremental_energy_savings_mwh": {
+        "type": "number",
+        "description": (
+            "energy savings in the given report year resulting from new participants "
+            "in existing demand response programs and all participants in new demand "
+            "response programs."
+        ),
+        "unit": "MWh",
```

Comment:
This field is from the `core_eia861__yearly_energy_efficiency ` table, the offspring table of the pre-2012 `core_eia861__yearly_demand_side_management_ee_dr ` table. We talked to EIA about whether we could or should merge these tables, and they said no. However, this column is suspiciously similar to the `energy_efficiency_incremental_effects_mwh` from the newer table. Maybe we should investigate whether they should have the same name at least? 

This also goes for the `incremental_peak_reduction_mw` and the `energy_efficiency_incremental_actual_peak_reduction_mw` columns.